imagetagger/imagetagger/images/views.py

```diff
@@ -743,7 +743,7 @@ def remove_image_set_tag(request) -> Response:
     tag.imagesets.remove(image_set)
     serializer = SetTagSerializer(tag)
     serializer_data = serializer.data
-    if not tag.imagesets.all():
+    if not tag.imagesets.all() and tag.name != 'test':
```

Comment:
Why is .all() used (in the existing code)? This should be switched to .exist() to prevent unnecessary fetching of objects from the database.